frontend/src/bundles/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -1,16 +1,136 @@
-import { Button } from '~/bundles/common/components/components.js';
+import {
+    type UserSignInRequestDto,
+    userSignInValidationSchema,
+} from 'shared/build/index.js';
+
+import {
+    Button,
+    Checkbox,
+    FormControl,
+    FormLabel,
+    Grid,
+    Input,
+    Link,
+    Typography,
+} from '~/bundles/common/components/components.js';
+import { getValidClassNames } from '~/bundles/common/helpers/helpers.js';
+import { useAppForm, useCallback } from '~/bundles/common/hooks/hooks.js';
+
+import { DEFAULT_SIGN_IN_PAYLOAD } from './constants/constants.js';
+import styles from './styles.module.scss';
 
 type Properties = {
-    onSubmit: () => void;
+    isTalent: boolean;
+    onSubmit: (payload: UserSignInRequestDto) => void;
 };
 
-const SignInForm: React.FC<Properties> = () => (
-    <>
-        <h1>Sign In</h1>
-        <form>
-            <Button label="Sign in" />
-        </form>
-    </>
-);
+const SignInForm: React.FC<Properties> = ({ onSubmit, isTalent = true }) => {
+    const { control, errors, handleSubmit } = useAppForm<UserSignInRequestDto>({
+        defaultValues: {
+            email: DEFAULT_SIGN_IN_PAYLOAD.email,
+            password: DEFAULT_SIGN_IN_PAYLOAD.password,
+        },
+        validationSchema: userSignInValidationSchema,
+    });
+
+    const handleFormSubmit = useCallback(
+        (event_: React.BaseSyntheticEvent): void => {
+            void handleSubmit(onSubmit)(event_);
+        },
+        [handleSubmit, onSubmit],
+    );
+
+    const sellingPointText = isTalent
```

Comment:
I think sellingPointText is useless. 
Because when someone sign-in, we don't know whether it's an employer or a talent.
It may be used to customize the sign-up page.

Reply:
removed